lib/db_helper/tasks.rb

```diff
@@ -67,19 +69,34 @@
 
     backup_db = default_db.gsub('_','-')
 
-    db = InputReader.get_string(prompt: "Database (#{backup_db}):", default_value: backup_db)
+    db = InputReader.get_string(prompt: "Database (#{backup_db}):",
+                                default_value: backup_db)
 
     default_backup_dir = 'latest'
-    backup_dir = InputReader.get_string(prompt: "Backup directory (#{default_backup_dir}):", default_value: default_backup_dir)
-    backup_db = InputReader.get_string(prompt: "Backup database (#{db}):", default_value: db)
+    backup_dir = InputReader.get_string(
+                   prompt: "Backup directory (#{default_backup_dir}):",
+                   default_value: default_backup_dir)
+    backup_db = InputReader.get_string(
+                   prompt: "Backup database (#{db}):",
+                   default_value: db)
     default_hot_backup_dir_base = File.join('','media','backup','mysql')
-    default_hot_backup_dir = File.join(default_hot_backup_dir_base, "#{backup_db}", "#{backup_dir}")
-    hot_backup_dir = InputReader.get_string(prompt: "Backup path (#{default_hot_backup_dir}):", default_value: default_hot_backup_dir)
-    raise "#{hot_backup_dir} is not a valid path" unless File.directory?(hot_backup_dir)
+    default_hot_backup_dir = File.join(default_hot_backup_dir_base, backup_db, backup_dir)
+    hot_backup_dir = InputReader.get_string(
+                       prompt: "Backup path (#{default_hot_backup_dir}):",
+                       default_value: default_hot_backup_dir)
+    unless File.directory?(hot_backup_dir)
+      raise "#{hot_backup_dir} is not a valid path"
+    end
 
-    sandbox_proc = Sys::ProcTable.ps.find { |p| p.comm == 'mysqld' && 
-      ( p.respond_to?(:cwd) ? p.cwd.to_s.include?(default_db) : p.cmdline.include?(default_db)
-      ) }
+    sandbox_proc = Sys::ProcTable.ps.find { |p|
+      p.comm == 'mysqld' && (
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.